README.md

```diff
@@ -0,0 +1,12 @@
+# NodeJS mentoring program
+
+## Prerequisites
+* node - ^12.18.3
+* yarn - ^1.21.1
+
+* `yarn start:1.1` - task 1.1
```

Comment:
Now this program (task 1.1) doesn't start for me:

```
D:\Mentoring_NodeJS\Module_1\Code-Review\nodejs-mentoring\node_modules\ts-node\src\index.ts:434
    return new TSError(diagnosticText, diagnosticCodes)
           
TSError: ⨯ Unable to compile TypeScript:
src/task1.1/index.ts:5:1 - error TS2580: Cannot find name 'process'. Do you need to install type definitions for node? Try 'npm i @types/node'.

5 process.stdin.pipe(reversePipe).pipe(process.stdout);
  ~~~~~~~
src/task1.1/index.ts:5:38 - error TS2580: Cannot find name 'process'. Do you need to install type definitions for node? Try 'npm i @types/node'.

5 process.stdin.pipe(reversePipe).pipe(process.stdout);
```
I run additionally "npm i @types/node" but it didn't help me.  Program should work out of the box (after npm i) without any additional actions
TS compilation passed successfully:
```
PS D:\Mentoring_NodeJS\Module_1\Code-Review\nodejs-mentoring> tsc --noEmit --diagnostics
Files:               99
Lines:            43803
Nodes:           191360
Identifiers:      68915
Symbols:          57493
Types:            17627
Instantiations:   17952
Memory used:    105606K
I/O read:         0.02s
I/O write:        0.00s
Parse time:       0.58s
Bind time:        0.32s
Check time:       1.44s
Emit time:        0.00s
Total time:       2.33s
```

Reply:
yep, in the last commit I added typeRoots for tsconfig. It is not required. Removed and now it works fine